### PR TITLE
Adding doc page for Tabs component

### PR DESCRIPTION
**Pull request by jsuarezgonz**

### CODE REVIEW

File: website/screens/components/tabs/specs/TabsSpecsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+        <Figure caption="Use a scroll indicator in scrollable tabs.">
+          <Image src={specsScrollable} alt="image" />
+        </Figure>
+        <HeadingLink level={3}>Notification tabs</HeadingLink>
+        <Figure caption="Notification badges are always positioned aligned with label/icon in 48px tab container and at top right of the 72px tab container.">
+          <Image src={specsNotification} alt="image" />
+        </Figure>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>States</HeadingLink>
```
Comment (Lisa159) [inline website/screens/components/tabs/specs/TabsSpecsPage.tsx:56]: It should be level 3.

File: website/screens/components/tabs/specs/TabsSpecsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+import Figure from "../../../common/Figure";
+import Code from "../../../common/Code";
+import DocFooter from "../../../common/DocFooter";
+
+const TabsSpecsPage = () => {
+  return (
+    <DxcStack gutter="xxxlarge">
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Specifications</HeadingLink>
+        <HeadingLink level={3}>Min / Max width</HeadingLink>
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/tabs/specs/TabsSpecsPage.tsx:35]: It should have DxcStack per level 3.

File: website/screens/components/tabs/specs/TabsSpecsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,462 @@
...
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>States</HeadingLink>
+        <DxcText as="p">
+          Tabs can get different states based on user interaction. States:
+          inactive, enabled, hover, pressed, focus and disabled.
+        </DxcText>
+        <Image src={statesTabs} alt="image" />
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Anatomy</HeadingLink>
```
Comment (Lisa159) [inline website/screens/components/tabs/specs/TabsSpecsPage.tsx:64]: It should be level 3.

File: website/screens/components/tabs/specs/TabsSpecsPage.tsx [hunk 4]
```diff
@@ -0,0 +1,462 @@
...
+          <DxcText>Active tab indicator</DxcText>
+          <DxcText>Inactive icon (Optional if there’s a label)</DxcText>
+          <DxcText>Inactive text label (Optional if there’s an icon)</DxcText>
+          <DxcText>Tab item</DxcText>
+          <DxcText>Divider</DxcText>
+        </DxcList>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Placement and alignment</HeadingLink>
+        <HeadingLink level={3}>Placement</HeadingLink>
```
Comment (Lisa159) [inline website/screens/components/tabs/specs/TabsSpecsPage.tsx:79]: It should be level 4.

File: website/screens/components/tabs/specs/TabsSpecsPage.tsx [hunk 5]
```diff
@@ -0,0 +1,462 @@
...
+          of the width of the screen.
+        </DxcText>
+        <Figure caption="Center horizontally in container is allowed when there is few tabs.">
+          <Image src={defaultPlacement} alt="image" />
+        </Figure>
+        <Figure caption="Use an indicator when there are elements hidden in scrollable tabs.">
+          <Image src={scrollablePlacement} alt="image" />
+        </Figure>
+        <DxcText as="p">
+          <strong>Container</strong>
```
Comment (Lisa159) [inline website/screens/components/tabs/specs/TabsSpecsPage.tsx:125]: Maybe it should be HeadingLink with level 5?

File: website/screens/components/tabs/specs/TabsSpecsPage.tsx [hunk 6]
```diff
@@ -0,0 +1,312 @@
...
+          technologies so they have the necessary information to efficiently
+          navigate the content.
+        </DxcText>
+        <DxcText as="p">
+          Content authors need to ensure the content that is added to the tab
+          panel is accessible. For example, if you add an image to the panel you
+          need to include alternative text to pass accessibility testing.
+        </DxcText>
+        <DxcText as="p">
+          <DxcLink
```
Comment (Lisa159) [inline website/screens/components/tabs/specs/TabsSpecsPage.tsx:248]: It should have newWindow prop to open in a new tab.

Comment (jsuarezgonz) [reply website/screens/components/tabs/specs/TabsSpecsPage.tsx:248]: Done

File: website/screens/components/tabs/specs/TabsSpecsPage.tsx (hunk 3 above)
Comment (Lisa159) [reply website/screens/components/tabs/specs/TabsSpecsPage.tsx:55]: No need. Closing comment.

File: website/screens/components/tabs/specs/TabsSpecsPage.tsx [hunk 7]
```diff
@@ -0,0 +1,462 @@
...
+          <DxcText>Active text label (Optional if there’s an icon)</DxcText>
+          <DxcText>Active tab indicator</DxcText>
+          <DxcText>Inactive icon (Optional if there’s a label)</DxcText>
+          <DxcText>Inactive text label (Optional if there’s an icon)</DxcText>
+          <DxcText>Tab item</DxcText>
+          <DxcText>Divider</DxcText>
+        </DxcList>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Placement and alignment</HeadingLink>
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/tabs/specs/TabsSpecsPage.tsx]: No need. Closing comment.

File: website/screens/components/tabs/specs/TabsSpecsPage.tsx (hunk 4 above)
Comment (Lisa159) [reply website/screens/components/tabs/specs/TabsSpecsPage.tsx]: No need. Closing comment.

File: website/screens/components/tabs/specs/TabsSpecsPage.tsx (hunk 5 above)
Comment (Lisa159) [reply website/screens/components/tabs/specs/TabsSpecsPage.tsx]: No need. Closing comment.

File: website/screens/components/tabs/specs/TabsSpecsPage.tsx [hunk 8]
```diff
@@ -0,0 +1,462 @@
...
+              <DxcText as="p">
+                <strong>Right</strong>: Don’t stack more than 4 fixed tabs.
+              </DxcText>
+            </>
+          }
+        >
+          <Image src={mobileNavigation} alt="image" />
+        </Figure>
+        <DxcText as="p">
+          <strong>Panel navigation</strong>
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/tabs/specs/TabsSpecsPage.tsx]: No need. Closing comment.

File: website/screens/components/tabs/specs/TabsSpecsPage.tsx [hunk 9]
```diff
@@ -0,0 +1,462 @@
...
+        </DxcText>
+        <Figure caption="When used for panel navigation tabs are placed in the top of the panel using all available width.">
+          <Image src={panelNavigation} alt="image" />
+        </Figure>
+        <HeadingLink level={3}>Alignment</HeadingLink>
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/tabs/specs/TabsSpecsPage.tsx]: No need. Closing comment.

File: website/screens/components/tabs/specs/TabsSpecsPage.tsx [hunk 10]
```diff
@@ -0,0 +1,462 @@
...
+              </DxcText>
+            </>
+          }
+        >
+          <Image src={tabsPlacement} alt="image" />
+        </Figure>
+
+        <HeadingLink level={4}>Desktop</HeadingLink>
+        <DxcText as="p">
+          <strong>Default</strong>
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/tabs/specs/TabsSpecsPage.tsx]: No need. Closing comment.

File: website/screens/components/tabs/specs/TabsSpecsPage.tsx [hunk 11]
```diff
@@ -0,0 +1,313 @@
...
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <Code>Left-arrow</Code>
+              </td>
+              <td>
+                Moves focus to the previous tab. If focus is on the first tab,
+                moves focus to the last tab. Optionally, activates the newly
+                focused tab
```
Comment (Lisa159) [inline website/screens/components/tabs/specs/TabsSpecsPage.tsx:292]: Dot in the end of the text.

File: website/screens/components/tabs/specs/TabsSpecsPage.tsx [hunk 12]
```diff
@@ -0,0 +1,313 @@
...
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <Code>Right-arrow</Code>
+              </td>
+              <td>
+                Moves focus to the next tab. If focus is on the last tab
+                element, moves focus to the first tab. Optionally, activates the
+                newly focused tab
```
Comment (Lisa159) [inline website/screens/components/tabs/specs/TabsSpecsPage.tsx:302]: Dot in the end of the text.

File: website/screens/components/tabs/usage/TabsUsagePage.tsx [hunk 13]
```diff
@@ -0,0 +1,234 @@
...
+        </DxcText>
+        <Figure
+          caption={
+            <>
+              <DxcText as="p">
+                <strong>Left</strong>: Main navigation tabs are place above the
+                content.
+              </DxcText>
+              <DxcText as="p">
+                <strong>Right</strong>: Don’t stack more than 4 fixed tabs.
```
Comment (Lisa159) [inline website/screens/components/tabs/usage/TabsUsagePage.tsx:161]: Maybe here to change the ’ to code.

File: website/screens/components/tabs/usage/TabsUsagePage.tsx [hunk 14]
```diff
@@ -0,0 +1,234 @@
...
+        <HeadingLink level={3}>Main navigation</HeadingLink>
+        <DxcText as="p">
+          Users can navigate between tabs by tapping a tab, or by performing a
+          swipe gesture over content in mobile devices.
+        </DxcText>
+        <HeadingLink level={3}>Panel navigation</HeadingLink>
+        <DxcText as="p">
+          Interacting with the tabs makes the content scrolls to the point that
+          is associated with that specific tab, while the tabs keep fixed at the
+          top of the container.{" "}
```
Comment (Lisa159) [inline website/screens/components/tabs/usage/TabsUsagePage.tsx:207]: Remove unused {" "}

File: website/screens/components/tabs/specs/TabsSpecsPage.tsx (hunk 11 above)
Comment (jsuarezgonz) [reply website/screens/components/tabs/specs/TabsSpecsPage.tsx]: Done

File: website/screens/components/tabs/specs/TabsSpecsPage.tsx (hunk 12 above)
Comment (jsuarezgonz) [reply website/screens/components/tabs/specs/TabsSpecsPage.tsx]: Done

File: website/screens/components/tabs/usage/TabsUsagePage.tsx [hunk 15]
```diff
@@ -0,0 +1,234 @@
...
+          </DxcText>
+          <DxcText>
+            The content should have the same level of hierarchy.
+          </DxcText>
+          <DxcText>
+            Tabs can contain icons and text. Text labels should be short and
+            have a clear relation to content.
+          </DxcText>
+        </DxcList>
+        <HeadingLink level={3}>Dont´s</HeadingLink>
```
[the remark this answers is not in the document]
Comment (jsuarezgonz) [reply website/screens/components/tabs/usage/TabsUsagePage.tsx]: Done

File: website/screens/components/tabs/usage/TabsUsagePage.tsx [hunk 16]
```diff
@@ -0,0 +1,234 @@
...
+const TabsUsagePage = () => {
+  return (
+    <DxcStack gutter="xxxlarge">
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Usage</HeadingLink>
+        <DxcText as="p">
+          Tabs organize and allow navigation between groups of content that are
+          related and at the same level of hierarchy.
+        </DxcText>
+        <HeadingLink level={3}>Do´s</HeadingLink>
```
[the remark this answers is not in the document]
Comment (jsuarezgonz) [reply website/screens/components/tabs/usage/TabsUsagePage.tsx]: Done

File: website/screens/components/tabs/usage/TabsUsagePage.tsx (hunk 13 above)
Comment (jsuarezgonz) [reply website/screens/components/tabs/usage/TabsUsagePage.tsx:161]: Done

File: website/screens/components/tabs/usage/TabsUsagePage.tsx (hunk 14 above)
Comment (jsuarezgonz) [reply website/screens/components/tabs/usage/TabsUsagePage.tsx]: Done

File: website/screens/components/tabs/usage/TabsUsagePage.tsx [hunk 17]
```diff
@@ -0,0 +1,234 @@
...
+              </DxcText>
+            </>
+          }
+        >
+          <Image src={contentUsageTabImage} alt="image" />
+        </Figure>
+        <HeadingLink level={3}>Typography</HeadingLink>
+        <DxcText as="p">
+          Avoid to use all caps for tab labels. ALL CAPS is rarely a good idea
+          because it’s harder to read.{" "}
```
[the remark this answers is not in the document]
Comment (jsuarezgonz) [reply website/screens/components/tabs/usage/TabsUsagePage.tsx]: Done

File: website/screens/components/tabs/usage/TabsUsagePage.tsx [hunk 18]
```diff
@@ -0,0 +1,234 @@
...
+        >
+          <Image src={contentUsageTabImage} alt="image" />
+        </Figure>
+        <HeadingLink level={3}>Typography</HeadingLink>
+        <DxcText as="p">
+          Avoid to use all caps for tab labels. ALL CAPS is rarely a good idea
+          because it’s harder to read.
+        </DxcText>
+        <Figure
+          caption="Is recommended to use sentence-case capitalization and semibold
```
Comment (Lisa159) [inline website/screens/components/tabs/usage/TabsUsagePage.tsx:81]: It should be used the subject: "It is recommended".

**Review by Lisa159 — APPROVED** [review]

**Pull request merged**